agora-specs/Spec/Proposal.hs

```diff
@@ -352,9 +337,116 @@ specs =
               )
               AdvanceProposal
               ( ScriptContext
-                  Proposal.advancePropsoalWithsStake
+                  Proposal.advanceProposalWithInvalidOutputStake
                   (Spending Proposal.proposalRef)
               )
           ]
+      , group
+          "unlocking"
+          $ map
+            ( \pc ->
+                group
+                  (show pc <> " proposals")
+                  [ group
+                      "legal"
+                      [ group
+                          "retract votes and unlock stake while voting"
+                          [ UnlockStake.mkProposalValidatorTestCase
+                              ( UnlockStake.UnlockStakeParameters
+                                  pc
+                                  UnlockStake.Voter
+                                  True
+                                  VotingReady
+                              )
+                              True
+                          ]
```

Comment:
Bit of a nitpick, but I find this extremely hard to read :cry: How can we make it nicer to read?